CUWesternBlister/CommunityReviews
# Simplify form and custom post creation

This allows the admin to add questions to forms and create new forms (when there is a tool to associate a new form with a category). Must follow the naming conventions for form fields outlined in the wiki documentation: [https://github.com/CUWesternBlister/CommunityReviews/wiki/New-Form-Creation-Admin-Input-Conventions](url)

To test, you must get the new ski review form from the page repo, or correct a form using the highlighted link above. 

Tests:
- Write a review and make sure the custom post creation is working with styling:
> Reviews to write:
- > Known brand and product
- > Known brand and unknown product
- > Unkown brand and product
- Make sure they appear in the BCR Flagged Reviews admin tool, and that you can approve and deny them. 
- Try making a question on an existing form following the linked documentation. See if it shows up on the review/custom post created in the right section of the review. 
- test form creation will come with a different pull request.
